src/common/http.hpp

```diff
@@ -40,13 +40,13 @@
 #include <common/log.hpp>
 
 namespace mender {
-namespace update {
+namespace common {
 namespace http_resumer {
 class DownloadResumerClient;
 class HeaderHandlerFunctor;
 class BodyHandlerFunctor;
 } // namespace http_resumer
-} // namespace update
+} // namespace common
 } // namespace mender
```

Comment:
Too bad we still need to keep this around.

Once we are moving code, what do you think of moving `mender::common::http_resumer` namespace to `mender::common::http::resumer`? It looks a bit more tidy and it follows for example the design of `events` and `events::io`.

It however will still require that we declare here in `http.hpp` the classes of the resumer, but we can at least remove these lines and move the declarations below inside `mender::common::http` as just:
```
namespace mender {
namespace common {
namespace http {

namespace resumer {
class DownloadResumerClient;
class HeaderHandlerFunctor;
class BodyHandlerFunctor;
} // namespace resumer

using namespace std;
...
```

At this point I am only thinking out loud, it is not a requirement